core/src/test/java/org/infinispan/persistence/support/AsyncStoreWithoutEvictionFunctionalTest.java

```diff
@@ -62,31 +59,25 @@ protected String parameters() {
       return "segmented-" + segmented;
    }
 
-   private EmbeddedCacheManager configureCacheManager(boolean async) throws Exception {
+   private EmbeddedCacheManager configureCacheManager() {
 
-      tmpDirectory = new File(CommonsTestingUtil.tmpDirectory(this.getClass()));
+      tmpDirectory =CommonsTestingUtil.tmpDirectory(this.getClass());
```

Comment:
Nitpick: I won't stop merging if not fixed yet.
```suggestion
      tmpDirectory = CommonsTestingUtil.tmpDirectory(this.getClass());
```